cypress/integration/home/header.spec.js

```diff
@@ -1,8 +1,26 @@
 /// <reference types="cypress" />
 
+function closeCookieBanner() {
+  cy.get(".osano-cm-window").contains("Accept").click();
+}
+
+describe("Cookie banner", () => {
+  beforeEach(() => {
+    cy.visit("/");
+  });
+
+it(`should save the cookie preferences`, () => {
+    closeCookieBanner();
+
+    cy.get(".osano-cm-window").should("not.be.visible");
+  });
+});
+
 describe("Header links", () => {
   beforeEach(() => {
     cy.visit("/").setCookie("locale", "de");
+
+    closeCookieBanner();
```

Comment:
suggestion: maybe we just put the closing here completely and don't check it as a separate `describe/it` block?

Reply:
Yes, we can do that. I pushed an updated version. My thought would have been to test if the cookie banner is working in general but I could also write a separate test file for that.